db/re/item_group_db.yml

```diff
@@ -31605,3 +31605,19 @@ Body:
             Rate: 360
           - Item: Critical_Stone_Bottom
             Rate: 360
+  - Group: KACHUA_SECRET_BOX
+    SubGroups:
+      - SubGroup: 0
+        List:
+          - Item: Cachua_Coupon
+      - SubGroup: 1
+        List:
+          - Item: Red_Potion
+            Rate: 4
+          - Item: Yellow_Potion
+            Rate: 3
+          - Item: White_Potion
+            Rate: 2
+          - Item: Blue_Potion
+            Rate: 1
+            Announced: true
```

Comment:
I tested it here and the ad is saying that it opened [Kachua's Secret Key] (item) the correct thing would be to announce the [Kachua's Secret Box] (npc)
Example Official:
![image](https://user-images.githubusercontent.com/98105181/201078288-4c9549d5-df9f-4443-a932-09593a192f02.png)

Reply:
Fixed. Please test it.